data-ingestion-service/src/main/java/gov/cdc/dataingestion/nbs/services/NbsRepositoryServiceProvider.java

```diff
@@ -151,6 +159,80 @@ public NbsInterfaceModel saveElrXmlMessage(String messageId, String xmlMsg, bool
 		return nbsInterfaceModel;
 	}
 
+	public NbsInterfaceModel savingElrXmlNbsInterfaceModelHelper(String xmlMsg, NbsInterfaceModel item) throws XmlConversionException {

```

Comment:
As long as these are thoroughly tested, then we good

Reply:
It is and I have to discuss with you on the same process tomorrow